app/views/pages/_form.html.erb

```diff
@@ -10,14 +10,14 @@
 <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
 
 <!-- Answer type -->
-<% answer_types= [
-  OpenStruct.new(value: 'single_line'),
-  OpenStruct.new(value: 'address'),
-  OpenStruct.new(value: 'date'),
-  OpenStruct.new(value: 'email'),
-  OpenStruct.new(value: 'national_insurance_number'),
-  OpenStruct.new(value: 'phone_number')
-]  %>
-
-<%= f.govuk_collection_radio_buttons :answer_type, answer_types, :value, small: true %>
+<% answer_types= %w[ single_line address date email national_insurance_number phone_number ]  %>
```

Comment:
I know this is WIP at the moment, but could we maybe move this to the controller action instead of in the view?

Reply:
yes - makes sense. I've moved it to an instance variable in the controller for now.